react/the_react_ecosystem/styling_react_applications.md

```diff
@@ -28,21 +28,25 @@ What if everything's already done for you? Styling, behavior, and accessibility
 
 [Material UI](https://mui.com/), [Radix](https://www.radix-ui.com/), and [Chakra UI](https://chakra-ui.com/) are worth a mention when talking about component libraries.
 
-<div class="lesson-note lesson-note--tip" markdown="1" >
+There are also icon component libraries like [lucide react](https://lucide.dev/guide/packages/lucide-react), which let you include icons in your project as components.
 
-For learning purposes throughout this course, we recommend that you avoid using CSS frameworks or component libraries, and instead implement your component's styling from scratch i.e. use [CSS Modules](#css-modules) or a [CSS-in-JS](#css-in-js) option.
+<div class="lesson-note lesson-note--warning" markdown="1" >
+
+For learning purposes throughout this course, we strongly recommend that you avoid using CSS frameworks or component libraries (using icon component libraries is fine), and instead implement your component's styling from scratch i.e. use [CSS Modules](#css-modules) or a [CSS-in-JS](#css-in-js) option.
 
 </div>
 
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 
-1. Read [CSS Modules documentation](https://github.com/css-modules/css-modules) and [How to style React components using CSS Modules](https://www.makeuseof.com/react-components-css-modules-style/)
+1. Read [CSS Modules documentation](https://github.com/css-modules/css-modules) and [How to style React components using CSS Modules](https://www.makeuseof.com/react-components-css-modules-style/).
+
+1. Read [CSS vs CSS-in-JS](https://blog.logrocket.com/css-vs-css-in-js/) and [a thorough analysis of CSS-in-JS](https://css-tricks.com/a-thorough-analysis-of-css-in-js/).
 
-1. Read [CSS vs CSS-in-JS](https://blog.logrocket.com/css-vs-css-in-js/) and [a thorough analysis of CSS-in-JS](https://css-tricks.com/a-thorough-analysis-of-css-in-js/)
+1. Skim through the [styled components documentation](https://styled-components.com/).
 
-1. Skim through the [styled components documentation](https://styled-components.com/)
+1. Check out the [lucide icon catalog](https://lucide.dev/icons/) to see what types of icons they have.
```

Comment:
We don't need this as an assignment step. If someone did want to check out the icon catalog (or one for any other library), they can definitely find it themselves, especially as the library is already linked. They can go from there. Making it an assignment would be out of the lesson's scope and intent.

Reply:
> We don't need this as an assignment step. If someone did want to check out the icon catalog (or one for any other library), they can definitely find it themselves, especially as the library is already linked. They can go from there. Making it an assignment would be out of the lesson's scope and intent.

Sure, I removed it!